common/src/main/java/org/apache/drill/common/util/GuavaPatcher.java

```diff
@@ -73,7 +73,8 @@ private static void patchStopwatch() {
 
       logger.info("Google's Stopwatch patched for old HBase Guava version.");
     } catch (Exception e) {
-      logger.warn("Unable to patch Guava classes.", e);
+      logger.warn("Unable to patch Guava classes: {}", e.getMessage());
```

Comment:
These calls can be wrapped in a small static function

Reply:
Thanks, updated.